main.go

```diff
@@ -367,13 +371,15 @@ func (cmd *InstallCmd) Run(ctx *Context) error {
 
 					// Reload the daemon to pick up the override.conf.
 					fmt.Printf("  Reloading service...")
-					if err := exec.Command("ssh", compute, "systemctl daemon-reload").Run(); err != nil {
+					cmd = exec.Command("ssh", compute, "systemctl daemon-reload")
+					if err := runCommand(ctx,cmd); err != nil {
```

Comment:
go fmt might correct the missing whitespace after ctx here

Reply:
It does.  thanks.